packages/main/src/SegmentedButtonItem.ts

```diff
@@ -30,63 +34,72 @@ import { SEGMENTEDBUTTONITEM_ARIA_DESCRIPTION } from "./generated/i18n/i18n-defa
  *
  * `import "@ui5/webcomponents/dist/SegmentedButtonItem.js";`
  * @constructor
- * @extends ToggleButton
- * @implements { ISegmentedButtonItem }
+ * @extends UI5Element
+ * @implements { ISegmentedButtonItem, IButton }
  * @public
  */
 @customElement({
 	tag: "ui5-segmented-button-item",
+	renderer: litRender,
 	template: SegmentedButtonItemTemplate,
+	styles: segmentedButtonItemCss,
 	dependencies: [Icon],
 })
-class SegmentedButtonItem extends ToggleButton implements ISegmentedButtonItem {
+class SegmentedButtonItem extends UI5Element implements IButton, ISegmentedButtonItem {
 	/**
-	 * **Note:** The property is inherited and not supported. If set, it won't take any effect.
-	 * @default "Default"
+	 * Defines whether the component is disabled.
+	 * A disabled component can't be selected or
+	 * focused, and it is not in the tab chain.
+	 * @default false
 	 * @public
 	 */
-	@property({ type: ButtonDesign, defaultValue: ButtonDesign.Default })
-	declare design: `${ButtonDesign}`;
+	@property({ type: Boolean })
+	disabled!: boolean;
 
 	/**
-	 * **Note:** The property is inherited and not supported. If set, it won't take any effect.
+	 * Determines whether the component is displayed as selected.
 	 * @default false
 	 * @public
 	 */
 	@property({ type: Boolean })
-	declare iconEnd: boolean;
+	selected!: boolean;
 
 	/**
-	 * **Note:** The property is inherited and not supported. If set, it won't take any effect.
-	 * @default false
+	 * Defines the tooltip of the component.
+	 *
+	 * **Note:** A tooltip attribute should be provided for icon-only buttons, in order to represent their exact meaning/function.
+	 * @default ""
 	 * @public
+	 * @since 1.2.0
 	 */
-	@property({ type: Boolean })
-	declare submits: boolean;
+	@property()
+	tooltip!: string;
 
 	/**
-	 * **Note:** The property is inherited and not supported. If set, it won't take any effect.
-	 * @default {}
+	 * Defines the icon, displayed as graphical element within the component.
+	 * The SAP-icons font provides numerous options.
+	 *
+	 * Example:
+	 * See all the available icons within the [Icon Explorer](https://sdk.openui5.org/test-resources/sap/m/demokit/iconExplorer/webapp/index.html).
+	 * @default ""
 	 * @public
 	 */
-    @property({ type: Object })
-    declare accessibilityAttributes: AccessibilityAttributes;
+	@property()
+	icon!: string;
 
 	/**
-	 * **Note:** The property is inherited and not supported. If set, it won't take any effect.
-	 * @default "Button"
-	 * @public
+	 * Indicates if the element is focusable
+	 * @private
 	 */
-	@property({ type: ButtonType, defaultValue: ButtonType.Button })
-	declare type: `${ButtonType}`;
+	@property({ type: Boolean })
```

Comment:
this property is always false. seems like it's redundant.

Reply:
It's a private property part of the IButton interface. We can make it optional in the interface, but otherwise we need implementation of it.

Reply:
Oh ... Sorry, missed that. Now, I understand why it's here.